README.md

```diff
@@ -1,11 +1,584 @@
+| `float[8]` | `pixdim`         | Grid spacings.                  |
+| `float`    | `vox_offset`     | Offset into .nii file           |
+| `float`    | `scl_slope`      | Data scaling: slope.            |
+| `float`    | `scl_inter`      | Data scaling: offset.           |
+| `short`    | `slice_end`      | Last slice index.               |
+| `char`     | `slice_code`     | Slice timing order.             |
+| `char`     | `xyzt_units`     | Units of `pixdim[1..4]`         |
+| `float`    | `cal_max`        | Max display intensity           |
+| `float`    | `cal_min`        | Min display intensity           |
+| `float`    | `slice_duration` | Time for 1 slice.               |
+| `float`    | `toffset`        | Time axis shift.                |
+| `int`      | `glmax`          | ~~UNUSED~~                      |
+| `int`      | `glmin`          | ~~UNUSED~~                      |
+| `char[80]` | `descrip`        | any text you like.              |
+| `char[24]` | `aux_file`       | auxiliary filename.             |
+| `short`    | `qform_code`     | `NIFTI_XFORM_*` code.           |
+| `short`    | `sform_code`     | `NIFTI_XFORM_*` code.           |
+| `float`    | `quatern_b`      | Quaternion b param.             |
+| `float`    | `quatern_c`      | Quaternion c param.             |
+| `float`    | `quatern_d`      | Quaternion d param.             |
+| `float`    | `qoffset_x`      | Quaternion x shift.             |
+| `float`    | `qoffset_y`      | Quaternion y shift.             |
+| `float`    | `qoffset_z`      | Quaternion z shift.             |
+| `float[4]` | `srow_x`         | 1st row affine transform.       |
+| `float[4]` | `srow_y`         | 2nd row affine transform.       |
+| `float[4]` | `srow_z`         | 3rd row affine transform.       |
+| `char[16]` | `intent_name`    | 'name' or meaning of data.      |
+| `char[4]`  | `magic`          | **MUST** be `"ni1\0"` or `"n+1\0"`. |
+
+
+### Table 4.2. Data types
+
+In Zarr, the byte order **MUST** be specified by prepending one of `{"|", "<", ">"}` to the data type string.
+| Data type    | NIfTI  | Zarr    |
+| ------------ | ------ | ------- |
+| `uint8`      | `2`    | <code>"&vert;u1"</code> |
+| `int16`      | `4`    | `"i2"`  |
+| `int32`      | `8`    | `"i4"`  |
+| `float32`    | `16`   | `"f4"`  |
+| `complex64`  | `32`   | `"c8"`  |
+| `float64`    | `64`   | `"f8"`  |
+| `rgb24`      | `128`  | <code>[["r", "&vert;u1"], ["g", "&vert;u1"], ["b", "&vert;u1"]]</code> |
+| `int8`       | `256`  | <code>"&vert;i1"</code> |
+| `uint16`     | `512`  | `"u2"`  |
+| `uint32`     | `768`  | `"u4"`  |
+| `int64`      | `1024` | `"i8"`  |
+| `uint64`     | `1280` | `"u8"`  |
+| `float128`   | `1536` | `"f16"` |
+| `complex128` | `1792` | `"c16"` |
+| `complex256` | `2048` | `"c32"` |
+| `rgba32`     | `2304` | <code>[["r", "&vert;u1"], ["g", "&vert;u1"], ["b", "&vert;u1"], ["a", "&vert;u1"]]</code> |
+| `bool`       | 🛑 unsupported! | <code>"&vert;b1"</code> |
```

Comment:
via [nifti1.h](https://nifti.nimh.nih.gov/pub/dist/src/niftilib/nifti1.h)

```c
#define DT_BINARY                  1     /* binary (1 bit/voxel)         */
```

```suggestion
| `bool`       | `1` | <code>"&vert;b1"</code> |
```

Not to say it's not a terrible idea to try to use it...